### ISSUE TITLE
`dialog` without accessible name is not warned by `require-accessible-name`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#658]
- Markuplint version: `3.0.1`
- Parser lang: HTML
- Node.js version: `16.16.0`
- OS: Windows 11

## Describe the bug

`dialog` element without accessible name is not warned by `require-accessible-name` rule

## Code Example

```html
<dialog></dialog>
```

## Steps To Reproduce

1. lint the code of  `Code Example`.

## The current behavior

No warnings.

## The expected behavior

 `require-accessible-name` rule warns it.

## Notes

- `require-accessible-name` rule warns the code below:

```html
<div role="dialog"></div>
```

- This issue may be related:
https://github.com/markuplint/vscode-markuplint/issues/15
